gef-ui/server/services/__tests__/api.test.js

```diff
@@ -367,3 +367,24 @@ describe('getAddressesByPostcode()', () => {
     await expect(api.getAddressesByPostcode({ postcode: 'invalid', userToken })).rejects.toThrowError('Invalid postcode');
   });
 });
+
+describe('getAmendment()', () => {
+  it('returns the correct response', async () => {
```

Comment:
can we put the response in the it statement as in {{HttpStatusCode.Ok}} `it('should return {{HttpStatusCode.Ok}} when a facility can be found`

Reply:
Sure, have changed